<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewControlPanelWidget</class>
 <widget class="QWidget" name="ViewControlPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>397</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="mLogoButton">
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>196</width>
        <height>255</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Camera</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="5" column="0">
        <widget class="QPushButton" name="__linkedZoom">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>16</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="__elevationSpin">
         <property name="minimum">
          <double>-179.990000000000009</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="__elevationLabel">
         <property name="text">
          <string>Elevation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Azimut</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="__StepMove">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="__azimuthSpin">
         <property name="minimum">
          <double>-179.990000000000009</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="__zoomSpin">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mLightPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>353</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Light</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Material</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Ambient</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="__LightDiffuseButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="__LightAmbientButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Diffuse</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="__LightSpecularButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Specular</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="__lightElevationSpin">
            <property name="minimum">
             <double>-179.990000000000009</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Azimut</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Elevation</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QDoubleSpinBox" name="__LightDistance">
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="__lightAzimuthSpin">
            <property name="minimum">
             <double>-179.990000000000009</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Distance</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>134</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Grids</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="__unitGrid">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Unit Size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="__sizeGrid"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Grid Size</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
